PubHub/PubHub.AdminPortal/Components/Models/BookForm.cs

```diff
@@ -5,15 +5,16 @@ namespace PubHub.AdminPortal.Components.Models
 {
     public class BookForm
     {
+#pragma warning disable IDE0305 // Simplify collection initialization
```

Comment:
I don't think this one is needed(?)

Reply:
Otherwise there will be two warnings when I set the `.ToArray()` 🤔

Reply:
What are the warning?

Reply:
I'm pretty sure it wants you to do this:

```csharp
AuthorIds = [.. AuthorIds],
GenreIds = [.. GenreIds],
```

Reply:
Yep, i changed it!